controllers/followersController.js

```diff
@@ -0,0 +1,121 @@
+import createError from 'http-errors';
+import models from '../models';
+import Response from '../helpers/responseHelper';
+
+/**
+ * Class handling followers operation
+ *
+ * @class FollowersController
+ */
+class FollowersController {
+  /**
+   * User followers handler
+   *
+   * @static
+   * @param {object} request - Express Request object
+   * @param {object} response - Express Response object
+   * @returns {object} Response object
+   * @param {Function} next - Express NextFunction
+   * @memberof FollowersController
+   */
+  static async follow(request, response, next) {
+    const { params: { username } } = request;
+
+    try {
+      const { followable, follower } = await FollowersController.validateFollowable(request);
+      await followable.addFollower(follower);
+
+      return Response.send(response, 200, [], `Successfully followed ${username}`);
```

Comment:
#### Nitpick
- Thought we were using constants for statuses?

Reply:
Thanks for pointing that, changes have been made.